js/epub-fetch/zip_resource_fetcher.js

```diff
@@ -11,7 +11,7 @@
 //  used to endorse or promote products derived from this software without specific
 //  prior written permission.
 
-define(['jquery', 'URIjs', './discover_content_type', 'zip-ext', 'readium_shared_js/helpers'], function ($, URI, ContentTypeDiscovery, zip, Helpers) {
```

Comment:
Why is `zip` removed? This should not be a global, it should be injected via `zip-ext` (which as the name implies, provides the necessary extensions).